app/(root)/layout.tsx

```diff
@@ -1,7 +1,15 @@
+import LeftSidebar from "@/components/shared/layout/LeftSidebar";
+// import RightSidebar from "@/components/shared/layout/RightSidebar";
```

Comment:
same here

Reply:
I guess i should not just add everything and commit it if its not ready, but I was trying to get to the routes which I had said I'd promise to get done.  I'll pull everything together over the weekend and start next week even.  i probably need to pomodoro this situation , i dont know if im rushing myself or what but i somehow feel like ive gotten behind but started ahead, but it probably has to do with me minterpreting what the first days slides meant and startin to do that , i thought the 1 2 3 4 5 6 7 featuress that were listed on day 1 was the order we were doing it.